source/reference/kubectl-minio-plugin/kubectl-minio-tenant-expand.rst

```diff
@@ -92,18 +96,33 @@ The command supports the following flags:
 .. mc-cmd:: --volumes
    :required:
 
-   The number of volumes in the new MinIO Tenant Pool. 
+   Mutually exclusive with :mc-cmd:`~kubectl minio tenant expand --volumes-per-server`.
+   Use either :mc-cmd:`~kubectl minio tenant expand --volumes` or :mc-cmd:`~kubectl minio tenant expand --volumes-per-server`.
+
+The number of volumes in the new MinIO Tenant Pool. 
    :mc:`kubectl minio` generates one Persistent Volume Claim (``PVC``) for each volume. 
 
-   The number of volumes affects both the requested storage of each ``PVC`` *and* the number of ``PVC`` to associate to each MinIO Pod in the new Pool:
+   The number of volumes affects both the requested storage of each ``PVC`` *and* the number of ``PVCs`` to associate to each MinIO Pod in the new Pool:
 
    - The command :mc:`kubectl minio` divides the :mc-cmd:`~kubectl minio tenant expand --capacity` by the number of volumes to determine the amount of ``resources.requests.storage`` to set for each ``PVC``.
 
-   - :mc:`kubectl minio` determines the number of ``PVC`` to associate to each ``minio`` server by dividing :mc-cmd:`~kubectl minio tenant expand --volumes` by :mc-cmd:`~kubectl minio tenant expand --servers`.
+   - :mc:`kubectl minio` determines the number of ``PVCs`` to associate to each ``minio`` server by dividing :mc-cmd:`~kubectl minio tenant expand --volumes` by :mc-cmd:`~kubectl minio tenant expand --servers`.
+
+   The command generates each ``PVC`` with Pod-specific selectors, such that each Pod only uses ``PVs`` that are locally-attached to the node running that Pod.
+
+   If the specified number of volumes exceeds the number of unbound ``PVs`` available in the cluster, :mc:`kubectl minio tenant expand` hangs and waits until the required ``PVs`` exist.
+
+.. mc-cmd:: --volumes-per-server
+   :required:
+
+   Mutually exclusive with :mc-cmd:`~kubectl minio tenant expand --volumes`.
+   Use either :mc-cmd:`~kubectl minio tenant expand --volumes-per-server` or :mc-cmd:`~kubectl minio tenant expand --volumes`.
+
+   Number of volumes to use for each server in the pool.
 
-   The command generates each ``PVC`` with Pod-specific selectors, such that each Pod only uses ``PV`` that are locally-attached to the node running that Pod.
+   Similar to :mc-cmd:`~kubectl minio tenant expand --volumes`, but instead of specifying the total number of volumes for all MinIO servers, associate ``--volumes-per-server`` volumes to each server.
```

Comment:
So where --volumes does a division (servers / volumes) to get volumes per server, this one does a multiplication (-server x volumes-per-server) to get the total number of volumes.

Right?

(And also on the create command.)

Reply:
Yup. Lemme see how I can clarify that more. It _is_ kinda confusing.

Reply:
Took out the math (in below paragraph you were referencing) and reworded. Left the existing `--volumes` text as it was.